### DISCUSSION TITLE
Hints section removed from readme

### DISCUSSION THREAD

**Discussion by Hootie81** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#419] [General]
Minor documentation issue that had me stumped for a bit

there is a link at several places in the readme still that don't work anymore, in the TOC and 4 other places
See [hints](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/README.md#hints) to check how to easily retrieve zone/point coordinates

looking at the history of the file, hints was removed on 17 Jan and replaced with the links to discussion 318. 

I think where the hints link is currently positioned stands out more than where the new link to the discussion is, perhaps the link could be updated to the discussion?

**Comment by PiotrMachowski** [comment]
Yeah, I have to update it

**Reply by Hootie81** [reply]
while your in there.. I'm struggling to define a zone as per the discussion 318

if I select a area and hold the play button I get
```
{
  "domain": "xiaomi_miot",
  "service": "call_action",
  "serviceData": {
    "entity_id": "vacuum.dreame_p2028_e5ea_robot_cleaner",
    "siid": 4,
    "aiid": 1,
    "params": [
      {
        "piid": 1,
        "value": 19
      },
      {
        "piid": 10,
        "value": "{\"areas\":[[731,5575,4758,10054,1,1,1]]}"
      }
    ],
    "force_params": true
  }
}
```
how do i put these values into the outline format?
this doesnt seem to work   [[ 731, 5575] , [4758,10054]]

**Reply by PiotrMachowski** [reply]
These are coordinates of opposite corners of a rectangular zone. To make an outline you need 4 points: [[731, 5575], [731, 10054], [4758, 10054], [4758, 5575]]
